integreat_cms/cms/utils/external_calendar_utils.py

```diff
@@ -302,7 +312,7 @@ def decode_by_day(self) -> list[int]:
         return weekdays
 
 
-def import_events(calendar: ExternalCalendar, logger: logging.Logger) -> None:
+def import_events(calendar: ExternalCalendar, logger: logging.Logger) -> ImportResult:
```

Comment:
Github won't let me make a suggestions, but the docstring should be adjusted to mention that this function now returns something